lib/shopify_app/sessions_concern.rb

```diff
@@ -87,5 +88,16 @@ def return_address
       session.delete(:return_to) || main_app.root_url
     end
 
+    def perform_after_authenticate_job
+      config = ShopifyApp.configuration.after_authenticate_job
+
+      return unless config && config[:job].present? && config[:inline].in?([true, false])
+
+      if config[:inline]
```

Comment:
Do we have a default value for `config[:inline]` ? If not, we won't get to this statement because most people won't set ` && config[:inline].in?([true, false])`. 

```
return unless config && config[:job].present?

if config[:inline] == true
...
```